src/styles/components/filter-bar.less

```diff
@@ -0,0 +1,116 @@
+.filter-bar {
+
+  display: flex;
+  flex-wrap: wrap-reverse;
+  width: 100%;
+
+  .filter-bar-left, .filter-bar-right {
+
+    display: inline-flex;
+    flex: 0 0 auto;
+    flex-wrap: wrap;
+    max-width: 100%;
+    width: auto;
+
+    .filter-bar-item {
+
+      flex: 0 0 auto;
+      max-width: 100%;
+
+    }
+
+  }
+
+  .filter-bar-right {
+
+    margin-left: auto;
+
+    &:last-child {
+
+      padding-right: 0;
+
+    }
+
+    .filter-bar-item:last-child {
+
+      // This overrides all of the specificity applied by the classes
+      // below.
+      padding-right: 0 !important;
```

Comment:
Sorry to have missed this the first time @orlandohohmeier, thanks for your patience and persistence! (: